Packages.props

```diff
@@ -27,8 +27,9 @@
 		<PackageReference Update="Microsoft.Azure.Kusto.Data" Version="9.0.4" />
 		<PackageReference Update="Microsoft.Azure.Kusto.Language" Version="9.0.4" />
 		<PackageReference Update="Microsoft.SqlServer.Assessment" Version="[1.1.9]" />
-		<PackageReference Update="Microsoft.SqlServer.Migration.Assessment" Version="1.0.20221014.16" />
-		<PackageReference Update="Microsoft.SqlServer.Management.SqlParser" Version="160.22519.0" />
+		<PackageReference Update="Microsoft.SqlServer.Migration.Assessment" Version="1.0.20220816.40" />
```

Comment:
maybe pull from latest main, `Microsoft.SqlServer.Migration.Assessment` should already be updated in my last PR: https://github.com/microsoft/sqltoolsservice/pull/1734

Reply:
(I see you already did that yesterday, I just merged mine this morning)

Reply:
done!